### PR TITLE
docs: add GDE profile of David Muellerchen

### PR DESCRIPTION
**Pull request by web-dave**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -823,5 +823,13 @@
     "picture": "hafiz.jpg",
     "website": "https://github.com/ayazhafiz",
     "bio": "Ayaz is an undergraduate student with interests in free software."
+  },
+  "webdave": {
+    "name": "David Muellerchen",
+    "picture": "webdave.jpg",
+    "twitter": "webdave_de",
+    "website": "https://github.com/web-dave",
+    "bio": "David is an Angular consultant who offers support for your Angular applications and projects. He’s also a trainer at angular.de, angular-college and swissangular. He loves to speak and teach at meetups and conferences. David also writes his own blog, rewiews upcoming books and articles. During leisure time he jumps arround with his 3 boys and his wife. He is a organizer of Angular Hamburg ng-de.",
```
Comment (kapunahelewong) [inline aio/content/marketing/contributors.json]: He is a organizer --> He is an organizer

And for this sentence:

> David also writes his own blog, rewiews upcoming books and articles.

Do you mean one of these?

> David also writes his own blog and reviews upcoming books and articles. (meaning that you review books and review articles)
OR
>David also writes his own blog, articles, and reviews upcoming books.(Meaning you write articles vs. review them)

rewiews --> reviews

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Thanks for this PR, @web-dave. Just a few thoughts.

**Review by StephenFluin — APPROVED** [review]

**Comment by kapunahelewong** [comment]
Hey @web-dave, will you tag me when you're ready? Also, could you squash so it's one commit? I'd like to help you move this through the process. Thanks!

**Review by kapunahelewong — APPROVED** [review]

**Comment by kapunahelewong** [comment]
merge assistance: global approval for docs changes

**Pull request closed without merging**